### ISSUE TITLE
API key stored in data.json, but not ignored via .gitignore

### ISSUE THREAD

**Issue by bachittle** [brianpetro/obsidian-smart-connections#299]
I track my obsidian notes with git, and noticed that I almost committed my OpenAI API key. This is not an issue for me personally as my obsidian vault is private, but for some open vaults (like for blogs) this may not be a good idea. I see that a gitignore is updated to ignore the smart connections folder. Could it be updated to also ignore this json?

Also a side tangent that could be made into a separate issue, but the chat histories are also blocked by the gitignore, but I think that is useful to sync, so I updated my personal gitignore to only block the embeddings, but not the chats. The data.json should definitely be ignored, but to ignore chats requires more debate.

**Comment by brianpetro** [comment]
Video response https://www.youtube.com/watch?v=ox2IAWrzRlA
